modular armour

[11:31:33] [WARN] Failed to handle packet for /92.125.0.240:63540
net.minecraft.util.ReportedException: Ticking player
 at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:302)
 at net.minecraft.network.NetHandlerPlayServer.func_147347_a(NetHandlerPlayServer.java:318)
 at net.minecraft.network.play.client.C03PacketPlayer.func_148833_a(C03PacketPlayer.java:34)
 at net.minecraft.network.play.client.C03PacketPlayer$C06PacketPlayerPosLook.func_148833_a(C03PacketPlayer.java:232)
 at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:231)
 at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
 at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:693)
 at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:335)
 at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:567)
 at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:444)
 at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:719)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/Minecraft
 at gregtech.common.items.armor.ModularArmor_Item.onArmorTick(ModularArmor_Item.java:341)
 at net.minecraft.entity.player.InventoryPlayer.func_70429_k(InventoryPlayer.java:319)
 at net.minecraft.entity.player.EntityPlayer.func_70636_d(EntityPlayer.java:541)
 at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1611)
 at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:291)
 at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:248)
 ... 10 more

--- a/src/main/java/gregtech/common/items/armor/ModularArmor_Item.java
+++ b/src/main/java/gregtech/common/items/armor/ModularArmor_Item.java
@@ -338,7 +338,7 @@ public class ModularArmor_Item extends ItemArmor implements ISpecialArmor, IGogg
 				if (stepup > 1) {
 					aPlayer.stepHeight = 1.0f;
 				}
-				if (GameSettings.isKeyDown(Minecraft.getMinecraft().gameSettings.keyBindJump)) {
+				if (aWorld.isRemote && GameSettings.isKeyDown(Minecraft.getMinecraft().gameSettings.keyBindJump)) {
 					if (stepup > 0 && jumpticks > 0) {
 						if (data.maxWeight > 2000) {
 							stepup *= 2000.0D / data.maxWeight;

